js/compiler/compiler.go

```diff
@@ -395,3 +413,27 @@ func (c *Pool) Get() *Compiler {
 func (c *Pool) Put(co *Compiler) {
 	c.c <- co
 }
+
+func verifySourceMapForBabel(srcMap []byte) error {
+	// this function exists to do what babel checks in sourcemap before we give it to it.
+	m := make(map[string]interface{})
```

Comment:
```suggestion
	m := make(map[string]json.RawMessage)
```
we don't really care about the values, so this should be more efficient, right/

Reply:
:shrug: dobut the performance of this will be interesting compared to the performance of parsing it by babel, or the source file by babel. But I guess it won't hurt :+1: